lib/possible_bipartition.rb

```diff
@@ -1,4 +1,37 @@
 
+
 def possible_bipartition(dislikes)
-  raise NotImplementedError, "possible_bipartition isn't implemented yet"
-end
+  
+
+  dislikes.length.times do |dog|
+    p dog
+    dog_groups = []
+    dog_groups[dog] = 1 #assign 1st dog to a group
+    queue = []
+    queue << dog
+
+    until queue.empty?
+      current = queue.shift #takes 1st element off the queue
```

Comment:
Just a note that this operation on an array is O(n), making this algorithm O(N^2 + E) where N is the number of nodes and E the number of edges.